test.sh

```
@@ -1,3 +1,3 @@
 cd test
 nosetests --with-coverage --cover-package pyecharts --cover-package test
-mv .coverage ../
```

This line hides the error from nosetests and all failing tests are hidden because mv succeeds and returns OK

---

got it.